prime-angular-frontend/src/app/modules/enrolment/pages/pharmanet-enrolment-summary/pharmanet-enrolment-summary.component.ts

```diff
@@ -186,12 +186,6 @@ export class PharmanetEnrolmentSummaryComponent extends BaseEnrolmentPage implem
   }
 
   public ngOnInit() {
-    // Only shown the first time the enrollee reaches the summary
```

Comment:
and remove the related progress bar variables that are no longer being used `this.showProgressBar`

Reply:
Removed.